bitrise/util.go

```diff
@@ -528,12 +528,6 @@ func removeStepDefaultsAndFillStepOutputs(stepListItem *models.StepListItemModel
 
 			hasOptions := false
 
-			if wfOptions.IsSensitive != nil && sOptions.IsSensitive != nil && *wfOptions.IsSensitive == *sOptions.IsSensitive {
```

Comment:
keep this section, it removes the redundant IsSensitive option from the bitrise.yml